Makefile

```diff
@@ -60,7 +60,7 @@ CONTAINER_ENGINE ?= docker
 
 BIN_DIR="bin"
 
-OPERATOR_SDK_VERSION="v1.25.0"
+OPERATOR_SDK_VERSION="v1.36.1"
```

Comment:
this depends on kube 1.29 so it seems it's fine: https://github.com/operator-framework/operator-sdk/releases/tag/v1.36.0